app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/acquisitiondetails/summary.scala.html

```diff
@@ -70,23 +74,32 @@
       )
     } else {
       @cyaRow(
-        messages("acquisitionDetails.cyaLabel.rebased"),
-        Html(messages("generic.yes")),
-        Some(cyaChange(messages("acquisitionDetails.cyaChange.rebasedAcquisitionPrice"), routes.AcquisitionDetailsController.rebasedAcquisitionPrice().url)),
-        "rebasedAcquisitionPrice"
-      )
-      @cyaRow(
-        messages("acquisitionDetails.cyaLabel.rebasedValue"),
+        messages("acquisitionDetails.cyaLabel.rebasedValue", LocalDateUtils.govShortDisplayFormat(cutoffDate)),
         Html(MoneyUtils.formatAmountOfMoneyWithPoundSign(rebasedValue.inPounds())),
         Some(cyaChange(messages("acquisitionDetails.cyaChange.rebasedAcquisitionPrice"), routes.AcquisitionDetailsController.rebasedAcquisitionPrice().url)),
         "rebasedAcquisitionPrice-value"
       )
     }
   }
-  @if(answers.improvementCosts.value === 0L){
+  @if(answers.shouldUseRebase){
```

Comment:
you don't need an outer if-else here - you could just have something like this:
```
cyaRow(
     messages("shouldUseRebase.title", LocalDateUtils.govShortDisplayFormat(cutoffDate)),
      Html(messages("generic.${if(shouldUseRebase) "yes" else "no"}")),  
     ....
    )
```
without the duplication